### PR TITLE
chore: Contribution File

### PR DESCRIPTION
**Pull request by hash-data**
# Description

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [ ] Scenario A
- [ ] Scenario B

# Screenshots or Recordings

### CODE REVIEW

**Comment by hanshal101** [comment]
Hey, this looks like good contribution guidelines, but I think we should also add about `signing-off` the PR before committing.
What's your thought on this?

**Comment by hash-data** [comment]
> Hey, this looks like good contribution guidelines, but I think we should also add about `signing-off` the PR before committing.

HI yes, that is what I have mentioned as a Good First Feature to add. 
do you want to contribute to it ?

**Comment by mrmagicpotato007** [comment]
@hash-data i would like to contribute to datazip, what is the roadmap?

**Comment by hash-data** [comment]
> @hash-data i would like to contribute to datazip, what is the roadmap?

Hi, @mrmagicpotato007, Thank you for your interest I would like to invite you to our slack channel to check out our roadmap, announcements, and community meeting on how to contribute : https://join.slack.com/t/getolake/shared_invite/zt-2usyz3i6r-8I8c9MtfcQUINQbR7vNtCQ

**Comment by hanshal101** [comment]
> > Hey, this looks like good contribution guidelines, but I think we should also add about `signing-off` the PR before committing.
> 
> HI yes, that is what I have mentioned as a Good First Feature to add. 
> do you want to contribute to it ?

Yup I would like to do that!

**Comment by piyushdatazip** [comment]
@hash-data Add Sign-Off requirements as must before raising PRs, PRs with unsigned commits won't be merged.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Submitting a pull request
+
+The type of change you make will dictate what repositories you will need to make pull requests for. You can reach out to us on our [**Slack**](https://join.slack.com/t/getolake/shared_invite/zt-2usyz3i6r-8I8c9MtfcQUINQbR7vNtCQ/) channel if you have any questions.
+
+For instance, to create a PR for contributing to a new driver, follow these instructions on [**submitting an Driver PR**]().
+
+## Committing
+
+We prefer squash or rebase commits so that all changes from a branch are committed to master as a single commit. All pull requests are squashed when merged, but rebasing prior to merge gives you better control over the commit message.
+
```
Comment (hash-data) [inline CONTRIBUTING.md:41]: suggestion: add example of conventional commit messages and conventional pr review comments

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+For any questions, concerns, or queries, you can start by asking a question on our [**Slack**](https://join.slack.com/t/getolake/shared_invite/zt-2usyz3i6r-8I8c9MtfcQUINQbR7vNtCQ) channel.
+<br><br>
+
+### We look forward to your feedback on improving this project!
+
+
+<!----variables---->
+
+[CLA]: https://datazip.io
```
Comment (hash-data) [inline CONTRIBUTING.md:56]: todo: update cla link

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Contributing to Olake
+
+Thanks for taking the time and for your help in improving this project!
+
+## Table of contents
+
+- [Olake Contributing Agreement](#olake-contributor-agreement)
+- [How You Can Contribute to Olake](#how-you-can-contribute-to-rudderstack)
```
Comment (shubham19may) [inline CONTRIBUTING.md]: rudderstack?

**Comment by hash-data** [comment]
Closing it as new PR https://github.com/datazip-inc/olake/pull/38 opened for the same

**Pull request closed without merging**